sdk/storage/azure-storage-blob/azure/storage/blob/_shared/uploads.py

```diff
@@ -336,11 +336,12 @@ def _upload_chunk(self, chunk_offset, chunk_data):
 class FileChunkUploader(_ChunkUploader):  # pylint: disable=abstract-method
 
     def _upload_chunk(self, chunk_offset, chunk_data):
-        chunk_end = chunk_offset + len(chunk_data) - 1
+        length = len(chunk_data)
+        chunk_end = chunk_offset + length - 1
         response = self.service.upload_range(
             chunk_data,
             chunk_offset,
-            chunk_end,
+            length,
```

Comment:
>, [](start = 18, length = 1)

was this a bug?

Reply:
This was fixed in file-shares _shared module, I'm just copying the changes across to sync the shared directory between the SDKs.
This line only affects the `FileChunkUploader` - so isn't used by Blobs